src/views/Error.vue

```diff
@@ -39,6 +41,19 @@
       </div>
       <!-- failsafe ends -->
 
+      <!-- TODO
+      remove hardcoded link -> either parameterize it, or enter a link to youtube -->
+      <div class="lead_text">
+        <p>Plio कैसे इस्तेमाल करना है, यह जानने के लिए यह video देखें </p>
+      </div>
+      <div class='embed-container'>
+        <iframe src='https://www.youtube.com/embed/FLOwzot27XM' frameborder='0' allowfullscreen>
```

Comment:
Agree that this should come as a parameter from somewhere.

Leave the TODO to change the constant, but at least put it as a property. When we want to change the URL, we wont' have to look for it deep in the code! This file is getting very big!